_15thnight/util.py

```diff
@@ -0,0 +1,44 @@
+import simplejson as json
+from flask import current_app
+from flask.json import dumps
+from flask.ext.login import current_user
+from functools import wraps
+
+
+class ExtensibleJSONEncoder(json.JSONEncoder):
+    """A JSON encoder that returns the to_json method if present"""
+    def default(self, obj):
+        if hasattr(obj, 'to_json'):
+            return obj.to_json()
+        return super(ExtensibleJSONEncoder, self).default(obj)
+
+def jsonify(*args, **kwargs):
```

Comment:
Well it looks like flask's `jsonify` has changed since I started using flask. It didn't used to support serializing kwargs directly to an object or serializing args into an array. 

It still doesn't support passing in args and kwargs. What this does with args and kwargs:

1. Checks if a single arg was passed in (line 24), if so, create a list with that single item in it. 
1. Otherwise, there were multiple args pass in and data is already a list, so it appends kwargs as a dictionary

`jsonify(1, 2, 3, hello="world", another="test")` would output `[1, 2, 3, { hello: "world", another="test"}]`.

We also can easily set the status code with a special `_status_code` kwarg.